tools/avatar-generator-GUI/main.py

```diff
@@ -0,0 +1,507 @@
+                 'WINTER_HAT4']
+    colors = ['BLACK', 'BLUE_01', 'BLUE_02', 'BLUE_03', 'GRAY_01', 'GRAY_02', 'HEATHER', 'PASTEL_BLUE', 'PASTEL_GREEN',
+              'PASTEL_ORANGE', 'PASTEL_RED', 'PASTEL_YELLOW', 'PINK', 'RED', 'WHITE']
+    expressions = ['CONCERNED', 'DEFAULT', 'DISBELIEF', 'EATING', 'GRIMACE', 'SAD', 'SCREAM_OPEN', 'SERIOUS', 'SMILE',
+                   'TONGUE',
+                   'TWINKLE', 'VOMIT']
+    eye_types = ['CLOSE', 'CRY', 'DEFAULT', 'DIZZY', 'EYE_ROLL', 'HAPPY', 'HEARTS', 'SIDE', 'SQUINT', 'SURPRISED',
+                 'WINK',
+                 'WINK_WACKY']
+    eyebrow_types = ['ANGRY', 'ANGRY_NATURAL', 'DEFAULT', 'DEFAULT_NATURAL', 'FLAT_NATURAL', 'FROWN_NATURAL',
+                     'RAISED_EXCITED',
+                     'RAISED_EXCITED_NATURAL', 'SAD_CONCERNED', 'SAD_CONCERNED_NATURAL', 'UNI_BROW_NATURAL', 'UP_DOWN',
+                     'UP_DOWN_NATURAL']
+    accessories_types = ['DEFAULT', 'KURT', 'PRESCRIPTION_01', 'PRESCRIPTION_02', 'ROUND', 'SUNGLASSES', 'WAYFARERS']
+    clothing_types = ['BLAZER_SHIRT', 'BLAZER_SWEATER', 'COLLAR_SWEATER', 'GRAPHIC_SHIRT', 'HOODIE', 'OVERALL',
+                      'SHIRT_CREW_NECK',
+                      'SHIRT_SCOOP_NECK', 'SHIRT_V_NECK']
+    graphic = ['BAT', 'BEAR', 'CUMBIA', 'DEER', 'DIAMOND', 'HOLA', 'PIZZA', 'RESIST', 'SELENA', 'SKULL',
+               'SKULL_OUTLINE']
+    emotion = ['neutral', 'speak', 'happy', 'sad', 'fear', 'angry', 'disgust', 'surprise']
+    return accessories_types, beard_styles, circle, clothing_types, colors, expressions, eye_types, eyebrow_types, facial_hair_colors, graphic, hair_colors, skin_tones, top_types, emotion
+
+
+def params_on_the_fly(values):
+    """ Parameters on-the-fly.
+
+        This function manages the parameters from values get from configuration and
+        output speak of the chatbot. And detect the I/O stream for sentimental
+        analysis to generate mouth movement and emotional expression.
+
+        : param values:
+        : return values:
+
+    """
+
+    # parameters on-the-fly
+    if values[0] is None:
+        Style = pa.AvatarStyle.CIRCLE
+
+    else:
+        if values[0] == 'CIRCLE':
+            Style = pa.AvatarStyle.CIRCLE
+        else:
+            Style = pa.AvatarStyle.TRANSPARENT
+
+    # SkinColor
+    if values[1] == 'BLACK':
+        Skin_color = pa.SkinColor.BLACK
+    elif values[1] == 'BROWN':
+        Skin_color = pa.SkinColor.BROWN
+    elif values[1] == 'DARK_BROWN':
+        Skin_color = pa.SkinColor.DARK_BROWN
+    elif values[1] == 'LIGHT':
+        Skin_color = pa.SkinColor.LIGHT
+    elif values[1] == 'PALE':
+        Skin_color = pa.SkinColor.PALE
+    elif values[1] == 'TANNED':
+        Skin_color = pa.SkinColor.TANNED
+    elif values[1] == 'YELLOW':
+        Skin_color = pa.SkinColor.YELLOW
+    else:
+        print("Invalid skin tone.")
+    # hair_colors
+    if values[2] == 'AUBURN':
+        Hair_color = pa.HairColor.AUBURN
```

Comment:
I would suggest to remove the code duplication by replacing the lengthy if/else statements with "getattr". Something like this:

hair_color_id = values[2]
hair_color = getattr(pa.HairColor, hair_color_id)